src/main/java/com/snowflake/kafka/connector/internal/SnowflakeSinkServiceV1.java

```diff
@@ -526,6 +515,26 @@ private void init(long recordOffset) {
       }
     }
 
+    private void initClientInfo() {
+      ConfigureClientResponse configureClientResponse = ingestionService.configureClient();
```

Comment:
A short description here would be very helpful. I know we usually dont put Javadocs on private method but it's just easier to follow the code. 

Also lets rename this to `initClientInfoForExactlyOnceDelivery`